shared/actions/platform-specific/index.native.tsx

```diff
@@ -64,7 +56,8 @@ const requestPermissionsToWrite = async () => {
 
 export const requestAudioPermission = async () => {
   let chargeForward = true
-  const {Permissions} = require('react-native-unimodules')
+  // TODO use expo-av etc and unify around that
+  const Permissions = require('expo-permissions')
```

Comment:
this moved to expo-permissions and then moved into the individual modules. later on we'll remove this and just use expo-av etc (and likely remove some other deps)